docs/source/cluster_management/sweep.rst

```diff
@@ -59,40 +59,33 @@ Note that some of these parameters are just used as a hint. Sweep dynamically mo
+   ``deleteBatchHint``, ``deleteBatchSize``, "128", "Target number of (cell, timestamp) pairs to delete in a single batch. Decrease if sweep cannot progress pass a large row or a large cell. Increasing it may improve sweep performance."
    ``pauseMillis``, "Only specified in config", "5000 ms", "Wait time between row batches. Set this if you want to use less shared DB resources, for example if you run sweep during user-facing hours."
 
-.. csv-table::
-   :header: "CasandraKeyValueService Config", "Endpoint Option", "Default", "Description"
-   :widths: 20, 20, 40, 200
-
-   ``timestampsGetterBatchSize``, "Only specified in config", "1,000", "Specify a limit on the maximum number of columns to fetch in a single database query. Set this to a number fewer than your number of columns if your Cassandra OOMs when attempting to run sweep with even a small row batch size. This parameter should be used when tuning Sweep for cells with many historical versions."
-
 Following is more information about when each of the batching parameters is useful.
 In short, the recommendation is:
 
 - Decrease ``candidateBatchHint`` and ``readLimit`` if there is memory pressure on the client.
-- Decrease ``timestampsGetterBatchSize`` if there is memory pressure on Cassandra even after setting ``candidateBatchHint`` down to 1.
+- Decrease ``candidateBatchHint`` if there is memory pressure on the KVS.
 
 Memory pressure on AtlasDB client: decrease candidateBatchHint and readLimit
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Since the number of columns per row can vary widely between tables, only setting a row batch size can lead to sweep batches containing different numbers of cells; setting ``candidateBatchHint`` can even this out.
-For example, consider a database that has one table with 1000 rows and 10 columns, a second table with 1000 rows and 100 columns, and that performs best when sweeping at most 10,000 cells at a time.
-In this setup, limiting ``candidateBatchHint`` to 100 would ensure that at most 10,000 cells are swept at a time, but would split the first table in 10 batches when only 1 was necessary.
-Instead, setting ``readLimit`` to 10,000 would allow both tables to be swept in optimal batches.
+``candidateBatchHint`` specifies the number of values to load from the KVS on each round-trip. ``readLimit`` specifies the number of values to read from the KVS in a batch.
+If each batch takes little time, it's advisable to increase the ``readLimit`` and ``deleteBatchHint``, to make sweep's batches bigger.
+If the client is OOMing, it's advisable to decrease the ``readLimit``, to have a lower number of values per batches.
+Since each batch contains at least ``candidateBatchHint`` values, if ``readLimit`` reaches the same value as ``candidateBatchHint`` you should reduce the latter config.
```

Comment:
nit: this line is hard to parse - might be good to mention ``candidateBatchHint`` a third time here.

It's also funny advise, since these values are identical by default.